relay-cardinality/src/redis/limiter.rs

```diff
@@ -210,9 +210,32 @@ impl QuotaScoping {
         self.namespace.is_none() || self.namespace == Some(entry.namespace)
     }
 
+    /// Returns the currently active slot.
+    pub fn active_slot(&self, timestamp: UnixTimestamp) -> Slot {
```

Comment:
I made the `window` field on the `RedisQuota` no longer public to prevent this.

Unfortunatly this does not work  for the `redis/limiter.rs` file, since it is the same module, I considered moving it to its own module to have "working" visibility modifiers.

I am not opposed to adding another new-type though.